src/bos/reporter/client.py

```diff
@@ -67,10 +68,19 @@ def get_auth_token(path='/opt/cray/auth-utils/bin/get-auth-token'):
             LOGGER.error(
                 'get_auth_token failed to retrieve authorization token: code=%d: error=%s',
                 e.returncode, e.output)
-        except Exception:
-            LOGGER.exception('Unexpected exception')
+        except Exception as e:
+            LOGGER.exception(f"Unexpected exception: {e}")
```

Comment:
pylint also dislikes the use of f-strings in calls to logging methods, preferring lazy evaluation, but I think that mainly matters for logging calls that may not end up in the log due to the logging level that is set -- and I suspect this one would always make it through that filter, so...

Reply:
Not sure if I should shift this back or not.